dist/index.html

```diff
@@ -1 +1 @@
\ No newline at end of file
+<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><meta http-equiv="X-UA-Compatible" content="ie=edge"><title>Todo App</title><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sacramento&display=swap&subset=latin-ext"><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"><link href="styles.css" rel="stylesheet"></head><body><div id="wrapper"><header><nav class="navbar navbar-light bg-success py-0"><div class="container-fluid px-0"><a class="navbar-brand" href="#"><i class="fa fa-cube d-inline-block text-white"></i> Todo App</a></div></nav></header><nav id="projects-nav" class="p-3 pt-5"><h4 class="d-flex flex-row justify-content-between"><span>Projects</span> <button class="btn mx-2 p-2 shadow-lg btn-show-form" data-toggle="collapse" data-target="#addProjectForm" type="button" aria-controls="addProjectForm" aria-expanded="false"><i class="fa fa-plus fa-fw"></i></button></h4><div class="bg-white collapse p-2 rounded-lg shadow-lg" id="addProjectForm"><form><div class="form-group"><span class="text-danger"></span></div><div class="form-group"><input class="form-control" id="name" placeholder="project name"></div><button type="submit" class="btn btn-block btn-primary btn-sm">Submit</button></form></div></nav><main class="p-5" id="app-container"><div class="row"><div class="d-flex flex-wrap w-100" id="todos-list"></div></div></main><nav id="key" class="bg-white shadow-lg p-3 pt-5"></nav></div><script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script><script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script><script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script><script src="runtime.0b47eaf22d00a80974d8.js"></script><script src="npm.css-loader.3ac90a1238623ffe59bf.js"></script><script src="main.039a182482bcf301959a.js"></script></body></html>
```

Comment:
For some reason, you're having two `index.html`, weird right?

Reply:
It's not weird, you should read about the HTMLWebpackPlugin and know that you can use a template to generate an index.html file in the dist folder. 
You can also refer to how Angular builds its own template - this is pretty standard and not a mistake.

Reply:
The same applies to the famous create-react-app. You should try ejecting one day and see what you will get.

Reply:
I don't you're to use `HTMLWebpackPlugin` or `Angular` for this project right?
Let me remind you;
![image](https://user-images.githubusercontent.com/31897434/74290216-79c65a00-4d31-11ea-82d1-84ce715b1260.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/
Kindly go back to the pathwright, and click on `previous lesson` webpack.

I am not seeing where to use `HTMLWebpackPlugin`, you can point me to it, maybe I am missing something.

Reply:
I am not using Angular - it's just a reference to larger projects that use the HTMLWebpack plugin and what they use it for.
The reason they taught us webpack is for us to use it for what it's used for, even bundling html files can be part of it. That's exactly what I used it for. Enlighten me here if I should not use it.